frontend/packages/console-app/src/components/user-preferences/UserPreferencePage.scss

```
@@ -0,0 +1,16 @@
+.co-user-preference-page-content {
+  display: flex;
+  flex-direction: row;
+  background-color: var(--pf-global--BackgroundColor--100);
+  height: 100%;
+  padding: 0 var(--pf-global--spacer--lg) var(--pf-global--spacer--lg) 0;
```

Instead of adding magic padding we should ensure that the tabs columns have the right width:

```suggestion
  &__tabs {
    flex: 0 0 220px; // similar to catalog
  }
```